app/controllers/api/databases.php

```diff
@@ -3235,7 +3235,7 @@ function updateAttribute(
 
         $collection = Authorization::skip(fn() => $dbForProject->getDocument('database_' . $database->getInternalId(), $collectionId));
 
-        if ($collection->isEmpty() || !$collection->getAttribute('enabled')) {
+        if ($collection->isEmpty() || (!$collection->getAttribute('enabled') && $mode !== APP_MODE_ADMIN)) {
```

Comment:
`Auth::isPrivilegedUser(Authorization::getRoles())` implies `$mode === APP_MODE_ADMIN` so I don't think we need both.

Reply:
I don't think it does, you can be a privileged user but not make the request in admin mode and vice versa

Reply:
`isPrivilegedUser()` returns true for Console users, right? I think a console user would only be detected if mode was admin:

https://github.com/appwrite/appwrite/blob/fcd4a69ad1b3db67c6d89f02c5f230c2038a4b35/app/init.php#L821-L824